crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -42,7 +42,7 @@ private KeyUtils() {
      * @param otherKeys An {@link Iterable} sequence of {@link PublicKey}.
      * @return True if <code>key</code> is in otherKeys.
      */
-    public static boolean isKeyInSet(@NotNull PublicKey key, @NotNull Iterable<PublicKey> otherKeys) {
+    public static boolean isKeyInKeys(@NotNull PublicKey key, @NotNull Iterable<PublicKey> otherKeys) {
```

Comment:
Would `containsKey` be a better name?

Reply:
If it was to be an extension function on `Collection<PublicKey>` then yes.